zkclient/welcome.go

```diff
@@ -169,22 +170,29 @@ func (ww *welcomeWindow) KeyHandler(w *ttk.Window, k ttk.Key) {
 					"Could not obtain remote identity")
 				return
 			}
-		} else {
-			pid = *ww.zkc.serverIdentity
-		}
 
-		ww.Status(w, false, "Connected to: %v %v", pid.Name,
-			pid.Fingerprint())
+			ww.Status(w, false, "Connected to: %v %v", pid.Name,
```

Comment:
whitespace

Reply:
gofmt disagrees, and we have to abide by gofmt or the Travis build fails.